### PR TITLE
✨ Add data table component

### PR DESCRIPTION
**Pull request by eliascodes**
* Adds generic `Card` component
* Adds `DataTable` component
  * Based on design in [design system](https://xd.adobe.com/spec/9bd1ae40-c6e9-41a3-6674-0cc8c430e9bb-ede0/screen/fb04cec4-41de-49d7-b6ef-73250d0644e2/Tables-and-charts/)
  * Sortable by any column
  * Uses `↑` and `↓` on column that is used for sorting
* Updated `FontSizeEnum` based on values in the design system

### CODE REVIEW

**Comment by eliascodes** [comment]
Default view (no scroll, overflow to the right):
Screenshot 2019-04-26 at 16 52 16

Scrolled a little to the right:
Screenshot 2019-04-26 at 16 52 26

**Comment by astroash** [comment]
@eliascodes is this awaiting review?

**Comment by eliascodes** [comment]
Yup, forgot to move it to the right column.

File: dashboard-app/src/styles/style_guide.ts [hunk 1]
```diff
@@ -3,19 +3,24 @@
...
   primary = '#8000FF',
   darkPrimary = '#5500aa',
   font = '#222222',
 }
 
 export enum FontSizeEnum {
-  heading = '1.6em',
-  heading2 = '1.2em',
-  medium = '0.8em',
+  heading = '2.75em',
```
Comment (astroash) [inline dashboard-app/src/styles/style_guide.ts]: Can font sizes be amended in another PR? This drastically changes the designs of other pages.

File: dashboard-app/src/components/Navbar/Navigation.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ const Link = styled(L)`
 
 const Navigation: React.FunctionComponent<Props> = (props) => (
   <Row between={'xs'}>
-    {['activity', 'volunteer', 'time'].map((page) =>
+    {['Activity', 'Volunteer', 'Time'].map((page) =>
```
Comment (astroash) [inline dashboard-app/src/components/Navbar/Navigation.tsx]: ⏪ this breaks the navigation...

We should also add a clientside 404 page at some point
